src/schema/organization/__tests__/expectedResults.js

```diff
@@ -1,3 +1,70 @@
+const organizationData =
```

Comment:
Using the script, keep only the more general data, that is `organizationData`, the script can do get particular fields